examples/rembi-metadata.yaml

```diff
@@ -48,8 +51,24 @@ acquisitions:
       ontology_name: EDAM
     imaging_instrument: High frequency ultrasound 5-element 40-MHz annular array
     image_acquisition_parameters: The dimensions of the HFU volumes vary from 150 x 161 x 81 to 210 x 281 x 282 voxels and the voxel size is 50 x 50 x 50 μm
+correlations:
+  "Image Correlation":
+    spatial_and_temporal_alignment: An alignment algorithm is used.
+    fiducials_used: Fiducials were randomly dropped into sample
+    transformation_matrix: Matrix is provided as a table.
+analysis:
+  "Image Analysis":
+    analysis_overview: "Analysis is done with custom tensorflow models. 
```

Comment:
I initially thought this wasn't valid yaml and wanted to suggest > but looks like double quotes are the most intuitive way of doing multiline strings (and escaping anything) https://yaml-multiline.info/

A comment along the lines of (below) might be a useful reference right next to where it would be useful?
Because both yaml and the output format are whitespace-sensitive, " is preferred for multi-line strings (replaces trailing whitespace and newline with a single space, escaped characters work as expected).

Reply:
Yes, I wasn't sure where to put it an explanation. Isabel asked me about putting line breaks in description and I've suggested using double quotes so wanted to test it and have it as an example somewhere. Maybe we can change it to one of the descriptions in the Study so it's as the start and put the explanation there.

Reply:
yep, anywhere would be nice, there are something like 3 different ways to do multiline strings, and " is the least weird (IMO)

Reply:
Actually this doesn't translate to a line break in pagetab which is what I wanted.